tests/nnstreamer_source/unittest_src_iio.cpp

```diff
@@ -540,13 +540,17 @@ build_dev_dir_scan_elements (iio_dev_dir_struct * iio_dev,
     signchar = 's';
     switch (idx % (iio_dev->num_scan_elements / 2)) {
       case 0:
+        /** big endian and signed */
         break;
       case 1:
+        /** little endian and unsigned (missing break is intended) */
         endianchar = 'l';
```

Comment:
plz add ```/* fallthrough */``` under this line so that svace tool passes this defect.

Reply:
updated.